### DISCUSSION TITLE
No active console user logged on, please connect and logon first.

### DISCUSSION THREAD

**Discussion by mahavatara** [rustdesk/rustdesk#5602] [Q&A] [answered]
I'm trying to connect from an M2 Mac to the latest headless Ubuntu Arm. I've read quite a few of the existing issues but don't seem to come anywhere close to resolving the above error message. I'm getting it when choosing "Transfer file" on the Mac, otherwise "Connect" seems to disconnect me as soon as I enter my password, I guess for the same reason.

I have "sessions galore":
SESSION  UID USER    SEAT  TTY  
      2 1001 user       pts/0
     c1  116 lightdm seat0 
     c2  116 lightdm

**Comment by SergeyMy** [comment]
why rustdesk when connecting to a non-GUI system?
there is SSH/SCP

**Reply by mahavatara** [reply]
why not?

**Reply by Ekcum** [reply]
sometimes u cant access the system via ssh for example if the server isnt in the same network and doesnt have a public ip

**Reply by Lilianne-Blaze** [reply]
Agree with Mahavatara, file transfer should still work even if the desktop doesn't

**Comment by uncommonthinker** [comment]
@mahavatara Were you able to find a solution for a non-gui based system?

**Comment by rustdesk** [comment, marked as answer]
Try login DE (desktop env) with non-root user. The Flatpak might not work with file transfer, because Flatpak runs in a sandbox, there is no permission to see file system on host.

**Reply by hardbrocklife** [reply]
What does this mean? Do you mean logging out on the linux machine make to sddm login? Is there a way to enable a logged in user from the desktop itself?

On a linux machine does rustdesk have access to the login screen like on windows?

I have Manjaro on all my machines, and cannot use the file transfer function between them because of this issue. I can I simulate or force a login that will satisfy rustdesk from a konsole after already being logged in?

loginctl says Im a session, but I get the same no user logged in error as OP:
![image](https://github.com/user-attachments/assets/173e3f28-1ade-45f5-830a-30a6e50e7b3f)

**Reply by Ekcum** [reply]
I solved this issue by deactivating wayland (on ubuntu desktop 24.04 lts) https://rustdesk.com/docs/en/client/linux/#login-screen

the op didnt have an issue like you. he tried connecting to a non-GUI OS

**Comment by fanybook** [comment]
@rustdesk 

![1741836457359](https://github.com/user-attachments/assets/873068fa-d6b1-4f2d-a42f-f86e931e7760)
![1741836680648](https://github.com/user-attachments/assets/239df144-5205-4e5d-8739-82e7646c2b48)


macos[12.6.2] click [File Transfer] not works（target device is archlinux）
archlinux click [File Transfer] can works（target device is macos[12.6.2]）

**Reply by AndryVo1** [reply]
@rustdesk
I have the same problem on a computer where rustdesk is installed via flatpak. But on the computer on which rustdesk is installed directly via .rpm, everything is working fine. 
Is there any way to fix this for flatpak?

**Comment by rustdesk** [comment]
Wondering if you can test this flatpak file? https://github.com/rustdesk/rustdesk/releases/download/nightly/rustdesk-1.4.0-x86_64.flatpak

**Reply by arcadematicas** [reply]
a mi me sigue dando el mismo error con esa version del paquete en flatpak. en anteriores versiones no tenia ese problema.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/commit/f919f297acae5768998c6dac30b69074738be8e5
